.gitignore

```diff
@@ -84,3 +84,6 @@ target
 
 examples/notebooks/*.html
 examples/notebooks/*.png
+
+# Visual Studio Code config files
+.vscode
```

Comment:
Just my opinion, but I think editor related stuff should go in the developers global ignore file, not in each project.

E.g. 

```
$ echo ".vscode" >> $HOME/.gitignore
$ git config --global core.excludesfile $HOME/.gitignore
```

This also holds for OS specific files like `DS_STORE` or `thumbs.db` or `desktop.ini`

Reply:
yes, that's fine. we can remove it.